cardano-cli/golden/files/golden/help/stake-address_drep-delegation-certificate.cli

```diff
@@ -0,0 +1,31 @@
+Usage: cardano-cli stake-address drep-delegation-certificate 
+            ( --stake-verification-key STRING
+            | --stake-verification-key-file FILE
+            | --stake-script-file FILE
+            | --stake-address ADDRESS
+            )
+            ( --drep-verification-key STRING
+            | --drep-cold-verification-key-file FILE
+            | --stake-pool-id STAKE_POOL_ID
+            )
+            --out-file FILE
+
+  Create a stake address drep delegation certificate
+
+Available options:
+  --stake-verification-key STRING
+                           Stake verification key (Bech32 or hex-encoded).
+  --stake-verification-key-file FILE
+                           Filepath of the staking verification key.
+  --stake-script-file FILE Filepath of the staking script.
+  --stake-address ADDRESS  Target stake address (bech32 format).
+  --drep-verification-key STRING
+                           DRep verification key (Bech32 or hex-encoded).
+  --drep-cold-verification-key-file FILE
+                           Filepath of the drep verification key.
+  --stake-pool-id STAKE_POOL_ID
```

Comment:
Shouldn't it be --drep-id?

Reply:
Since there are [2 pre-defined Dreps](https://github.com/cardano-foundation/CIPs/blob/e96681908fd336702b9d89827063a190c5656eab/CIP-1694/README.md#pre-defined-dreps) (abstain and no-confidence),  we could have `--drep-abstain` and `--drep-no-confidence` as alternatives to `--drep-id`

Reply:
Will these have predefined hashes as well?